src/Classes/PartyTab.lua

```diff
@@ -0,0 +1,620 @@
+	self.controls.enemyMods = new("EditControl", {"TOPLEFT",self.controls.enemyModsLabel,"BOTTOMLEFT"}, 0, 2, 0, 0, "", nil, "^%C\t\n", nil, nil, 14, true)
+	self.controls.enemyMods.width = function()
+		return self.width / 2 - 16
+	end
+	self.controls.enemyMods.height = function()
+		return (self.controls.enemyMods.hasFocus and (self.height - 304) or 106)
+	end
+
+	self.controls.editCursesLabel = new("LabelControl", {"TOPLEFT",self.controls.enemyMods,"BOTTOMLEFT"}, 0, 8, 150, 16, "^7Curses")
+	self.controls.editCurses = new("EditControl", {"TOPLEFT",self.controls.editCursesLabel,"BOTTOMLEFT"}, 0, 2, 0, 0, "", nil, "^%C\t\n", nil, nil, 14, true)
+	self.controls.editCurses.width = function()
+		return self.width / 2 - 16
+	end
+	self.controls.editCurses.height = function()
+		return ((not self.controls.enemyCond.hasFocus and not self.controls.enemyMods.hasFocus) and (self.height - 304) or 106)
+	end
+	self:SelectControl(self.controls.editAuras)
+end)
+
+function PartyTabClass:Load(xml, fileName)
+	for _, node in ipairs(xml) do
+		if node.elem == "ImportedText" then
+			if not node.attrib.name then
+				ConPrintf("missing name")
+			elseif node.attrib.name == "Aura" then
+				self.controls.editAuras:SetText(node.attrib.string)
+				self:ParseBuffs(self.processedInput["Aura"], node.attrib.string, "Aura")
+			elseif node.attrib.name == "Curse" then
+				self.controls.editCurses:SetText(node.attrib.string)
+				self:ParseBuffs(self.processedInput["Curse"], node.attrib.string, "Curse")
+			elseif node.attrib.name == "EnemyConditions" then
+				self.controls.enemyCond:SetText(node.attrib.string)
+				self:ParseBuffs(self.enemyModList, node.attrib.string, "EnemyConditions")
+			elseif node.attrib.name == "EnemyMods" then
+				self.controls.enemyMods:SetText(node.attrib.string)
+				self:ParseBuffs(self.enemyModList, node.attrib.string, "EnemyMods")
+			end
+		elseif node.elem == "ExportedBuffs" then
+			if not node.attrib.name then
+				ConPrintf("missing name")
+			end
+			if node.attrib.name ~= "EnemyConditions" and node.attrib.name ~= "EnemyMods" then
+				self:ParseBuffs(self.buffExports, node.attrib.string, node.attrib.name)
+			end
+			--self:ParseBuffs(self.buffExports, node.attrib.string, "Aura")
+			--self:ParseBuffs(self.buffExports, node.attrib.string, "Curse")
+			--self:ParseBuffs(self.buffExports, node.attrib.string, "EnemyConditions")
+			--self:ParseBuffs(self.buffExports, node.attrib.string, "EnemyMods")
+		end
+	end
+	self.lastContentAura = self.controls.editAuras.buf
+	self.lastContentCurse = self.controls.editCurses.buf
+	self.lastContentEnemyCond = self.controls.enemyCond.buf
+	self.lastContentEnemyMods = self.controls.enemyMods.buf
+	self.lastEnableExportBuffs = self.enableExportBuffs
+end
+
+function PartyTabClass:Save(xml)
+	local child = { elem = "ImportedText", attrib = { name = "Aura" } }
+	child.attrib.string = self.controls.editAuras.buf
```

Comment:
Instead of setting a string attribute like this, just insert the text directly into the element, like how Items are stored.

Reply:
that doesnt save too much space and does require a table inset but I can see about doing this to make it more readable